hazelcast/src/test/java/com/hazelcast/map/BasicMapTest.java

```diff
@@ -875,6 +877,29 @@ public void testEntryView() {
                 MILLISECONDS.toSeconds(time1), MILLISECONDS.toSeconds(time2));
     }
 
+    @Test
+    public void testEntryViewLastUpdateTime() {
```

Comment:
seems this can be divided into 2 tests testing two assertions separately. 
Also please include test's goal inside test method name like 
`test_non_expirable_entry_lastUpdateTime_is_unset_when_per_entry_stats_disabled`
`test_expirable_entry_lastUpdateTime_is_set_when_per_entry_stats_disabled`

Reply:
https://github.com/hazelcast/hazelcast/pull/23279/commits/8b754a4fedef011b1c2ec33e6f9b889bdfbc89c8